types/office-js-preview/index.d.ts

```diff
@@ -12082,11 +12082,13 @@ declare namespace Office {
          * {@link https://docs.microsoft.com/office/dev/add-ins/outlook/outlook-on-send-addins?tabs=windows | on-send feature using `ItemSend` in the manifest},
          * append-on-send runs before on-send functionality.
          *
+         * **Important**: To use `appendOnSendAsync`, the `ExtendedPermissions` manifest node must include the `AppendOnSend` extended permission. To learn more
```

Comment:
Consider moving this before the info about ItemSend. That way, the two sets of info related to ItemSend stay together